packaging/package_aimet.cmake

```diff
@@ -192,7 +192,11 @@ foreach(package ${package_name_list})
       COMMAND find ${build_packaging_dir} -name "libaimet_tf_ops*.so" -exec ${PATCHELF_EXE} --set-rpath $ORIGIN:$ORIGIN/../../tensorflow:$ORIGIN/../../tensorflow/python {} \;
   )
   # Invoke the setup tools script to create the wheel packages.
-  execute_process(COMMAND python3 setup.py sdist bdist_wheel ${CUDA_OPTION} WORKING_DIRECTORY ${build_packaging_dir} OUTPUT_VARIABLE output_var)
+  execute_process(
+    COMMAND ${Python3_EXECUTABLE} setup.py sdist bdist_wheel ${CUDA_OPTION}
```

Comment:
Change to "PYTHON3_EXECUTABLE" if you make the above requested change

Reply:
yes, of course.